config/vufind/EDS.ini

```diff
@@ -279,6 +279,12 @@ password  = "PASSWORD"
 ;password_file = /path/to/secret
 profile   = "PROFILE"
 organization_id = "VuFind from MyUniversity"
+; EBSCO might issue you with an API Key, fill in below if it was issued
+; depending on the key vended to you (legacy / new) this API key might be required
+; for successful API queries
+api_key = ""
```

Comment:
I think the context of these keys is complex enough that EDS.ini needs more of this info you shared in the PR description, or else maybe a link to an 'EDS' page on the VuFind wiki where it's explained in full.

> API keys are used to monitor access to EDS API, isolating traffic by customer and use-case. This isolation allows EBSCO to manage performance for specific keys (use cases). Should a request be throttled, the gateway will respond with an HTTP 429.
Existing implementations may start using API keys at any time. EBSCO will require API keys for all customers at one point, but deadlines have not been established. Customers may have more than one API key for separate client applications or use-cases accessing EDS API.
Note: API keys do not replace EDS API’s authentication or session tokens.